Error when multiple packages import the target `console_bridge::console_bridge`

Apparantly for some packages you need to wrap [lines 5 to 11](https://github.com/rock-core/base-console_bridge/blob/master/src/console_bridgeConfig.cmake#L5) into a conditional, to prevent multiple packages importing / creating the same target. Something like this seems to do the trick. 

    if (NOT TARGET console_bridge::console_bridge)
    ...
    endif() 

Not sure if there are more elegant approaches though.

Some combinations seem to work without a problem though (e.g., `control/urdfdom` and `tools/class_loader`). Not sure what exactly is going on there.

The error message:

    CMake Error at /path/to/ws/install/base/console_bridge/share/console_bridge/cmake/console_bridgeConfig.cmake:5 (add_library):
      add_library cannot create imported target "console_bridge::console_bridge"
      because another target with the same name already exists.

--- a/src/console_bridgeConfig.cmake
+++ b/src/console_bridgeConfig.cmake
@@ -1,11 +1,13 @@
-find_package(Rock)
-include(FindPkgConfig)
-rock_find_pkgconfig(console_bridge REQUIRED console_bridge)
+if (NOT TARGET console_bridge::console_bridge)
+    find_package(Rock)
+    include(FindPkgConfig)
+    rock_find_pkgconfig(console_bridge REQUIRED console_bridge)
 
-add_library(console_bridge::console_bridge INTERFACE IMPORTED GLOBAL)
-set_target_properties(console_bridge::console_bridge PROPERTIES
-    INTERFACE_INCLUDE_DIRECTORIES "${console_bridge_INCLUDE_DIRS}"
-)
+    add_library(console_bridge::console_bridge INTERFACE IMPORTED GLOBAL)
+    set_target_properties(console_bridge::console_bridge PROPERTIES
+        INTERFACE_INCLUDE_DIRECTORIES "${console_bridge_INCLUDE_DIRS}"
+    )
 
-target_link_libraries(console_bridge::console_bridge INTERFACE
-    ${console_bridge_LIBRARIES})
+    target_link_libraries(console_bridge::console_bridge INTERFACE
+        ${console_bridge_LIBRARIES})
+endif()

